website/editor/src/evaluator/index.tsx

```diff
@@ -276,8 +290,15 @@ const removeImportsPlugin = babel => ({
         case 'effector-react/ssr':
           replaceModuleImports('effectorReactSSR', path, babel)
           break
+        case 'patronum':
+          replaceModuleImports('patronum', path, babel)
+          break;
         default:
-          path.remove()
+          // TODO: import actual module after patronum refactoring to ES Modules
```

Comment:
I think we can avoid additional modules altogether, reusing them like that is indeed a good thing